__tests__/endpoints.test.js

```diff
@@ -89,4 +89,24 @@ describe('receiving a 404 on an invalid param', () => {
             expect(body.message).toEqual('404 not found')
         })
     });
+});
+describe('GET /api/categoires', () => {
+        test('Return status:200 and array of review object ', () => {
+            //Arrange
+            //Act
+            return request(app).get('/api/reviews')
+            .expect(200)
+            .then((response) => {
```

Comment:
You can use a forEach here and test that each review in the array of objects has these properties - it's a small change that will make your test more rigorous.

You should also check the length of the returned array of objects too - to make sure you're getting the correct number of reviews back.

As the ticket asks for a default order and sort_by - it would be a good idea to use jest-sorted to check the sorting of the array too.